envoy-filters/mgw-source/filters/http/mgw-wasm-websocket/filter.cc

```diff
@@ -64,13 +64,18 @@ bool MgwWebSocketRootContext::onConfigure(size_t config_size) {
   return true;
 }
 
+MgwWebSocketContext::~MgwWebSocketContext(){
+  LOG_TRACE(std::string("MgwContext destructed") + this->x_request_id_);
```

Comment:
Just a query; Don't we have to delete stream_handler_ here?

Reply:
Nope 
calling reset on stream_handler deletes the object. (I verified it with logs). Deleting explicitly leads to issues and not necessary.